modules/aws_ecs/main.tf

```diff
@@ -435,6 +435,7 @@ resource "aws_ecs_task_definition" "retool_code_executor" {
         image     = local.ecs_code_executor_image
         cpu       = var.launch_type == "EC2" ? var.ecs_task_resource_map["code_executor"]["cpu"] : null
         memory    = var.launch_type == "EC2" ? var.ecs_task_resource_map["code_executor"]["memory"] : null
+        user      = "1001:1001"
```

Comment:
this is expected user if `CONTAINER_UNPRIVILEGED_MODE` is true

Reply:
should this only be set when `var.launch_type == 'FARGATE`, since we can support privileged mode for ECS on EC2? (same for setting envvar below)

Reply:
Ah taking another look.. what's with aws_ecs_ec2 folder ? should these just be one directory w/conditionals to differentiate?

Reply:
yeah every other module in this repo is ~deprecated. this aws_ecs module should support both EC2 and Fargate deployments going forward

Reply:
👍  I'll add separate PR to just remove that.